cmd/postcli/main.go

```diff
@@ -121,15 +126,32 @@ func main() {
 		return
 	}
 
-	if err := processFlags(); err != nil {
-		log.Fatalln("failed to process flags", err)
-	}
-
-	zapLog, err := zap.NewProduction()
+	zapCfg := zap.Config{
+		Level:    zap.NewAtomicLevelAt(logLevel),
+		Encoding: "console",
+		EncoderConfig: zapcore.EncoderConfig{
+			TimeKey:        "T",
+			LevelKey:       "L",
+			NameKey:        "N",
+			MessageKey:     "M",
+			LineEnding:     zapcore.DefaultLineEnding,
+			EncodeLevel:    zapcore.CapitalLevelEncoder,
+			EncodeTime:     zapcore.ISO8601TimeEncoder,
+			EncodeDuration: zapcore.StringDurationEncoder,
+		},
+		OutputPaths:      []string{"stdout"},
+		ErrorOutputPaths: []string{"stderr"},
+	}
+
+	zapLog, err := zapCfg.Build()
```

Comment:
I assume the goal is to have human readable logs with the specified level? Would this work:

```go
	zapCfg := zap.NewDevelopmentConfig()
	zapCfg.Level = zap.NewAtomicLevelAt(logLevel)
	zapLog, err := zapCfg.Build()
```

Reply:
It's actually a slightly tweaked Development preset with a few changes:
- don't print the file and line where a log was emitted,
- don't print stack on ERROR level log,
- print to stdout instead of stderr